blocks/blog-home/blog-home.js

```diff
@@ -358,29 +337,22 @@ export default async function decorate(block) {
   const topics = new Set();
   const audiences = new Set();
   if (blogList.length) {
-    const blogContent = document.createElement('div');
-    blogContent.classList.add('blog-content');
+    const blogContent = createTag('div', { class: 'blog-content' });
     // Get default content in this section and add it to blog-content
     const defaultContent = document.querySelectorAll('.blog-home-container > .default-content-wrapper');
     defaultContent.forEach((div) => blogContent.append(div));
     // Create the selected filters DOM structure
-    const selectedFilters = document.createElement('div');
-    selectedFilters.classList.add('selected-filters');
-    const selectedFiltersdiv = document.createElement('div');
-    selectedFiltersdiv.classList.add('selected-filters-div');
-    const selectedFiltersTitle = document.createElement('div');
-    selectedFiltersTitle.classList.add('selected-filters-title');
-    const clearAllFilters = document.createElement('div');
-    clearAllFilters.classList.add('clear-all-filters');
-    const selectedFiltersList = document.createElement('div');
-    selectedFiltersList.classList.add('selected-filters-list');
+    const selectedFilters = createTag('div', { class: 'selected-filters' });
+    const selectedFiltersdiv = createTag('div', { class: 'selected-filters-div' });
+    const selectedFiltersTitle = createTag('div', { class: 'selected-filters-title' });
+    const clearAllFilters = createTag('div', { class: 'clear-all-filters' });
+    const selectedFiltersList = createTag('div', { class: 'selected-filters-list' });
```

Comment:
Nitpicking: Might be good to consolidate these multiple variable declarations into a single line.

```suggestion
    const selectedFilters = createTag('div', { class: 'selected-filters' }),
      selectedFiltersdiv = createTag('div', { class: 'selected-filters-div' }),
      selectedFiltersTitle = createTag('div', { class: 'selected-filters-title' }),
      clearAllFilters = createTag('div', { class: 'clear-all-filters' }),
      selectedFiltersList = createTag('div', { class: 'selected-filters-list' });
```

Reply:
Yes, of course -- thank you.